### DISCUSSION TITLE
yyg/5ea27f40/

### DISCUSSION THREAD

**Discussion by giscus** [MarsperL/MarsperL.github.io#22] [Announcements]
# yyg/5ea27f40/

在Github上使用OsmosFeed搭建在线RSS阅读器（无需服务器）前言在搭建云野阁博客网站期间，看了好多具有个人特色的博客网站，有不少都有RSS订阅功能，那RSS是什么呢？咱们先唠唠。  🔜RSS🔚                               RSS 全称 Really Simple Syndication（真正简易联合），是一种基于XML（可扩展标记语言）的内容分发协议

https://marsperl.github.io/yyg/5ea27f40/

**Comment by lzh1230** [comment]
支持绑定域名吗

**Reply by MarsperL** [reply]
OsmosFeed本身不支持，可以给github pages绑定域名
